spa/tsconfig.json

```diff
@@ -22,5 +23,5 @@
 			"rest-interfaces": ["./src/rest-interfaces"]
 		}
 	},
-  "include": ["src"]
+  "includes": [ "./setup.ts", "src" ]
```

Comment:
Otherwise doesn't looks like the ts-jest is including the tesing-lib/js-dom thing during the ts compile phase.

Reply:
I mean after I remove `import "@testing-library/jest-dom";` tests failed, turns out that `import "@testing-library/jest-dom";` is not called by ts-jest during compile.

ps: is has nothing to do with jest at compile time.